app/controllers/admin/messages.js

```diff
@@ -4,10 +4,15 @@ export default Controller.extend({
 
   actions: {
     save() {
-      let systemMessages = this.get('model');
-      systemMessages.forEach(systemMessage => {
-        systemMessage.save();
-      });
+      try {
+        let systemMessages = this.get('model');
+        systemMessages.forEach(systemMessage => {
+          systemMessage.save();
+        });
+        this.get('notify').success(this.get('l10n').t('Changes have been saved successfully'));
+      } catch (e) {
+        this.get('notify').error(this.get('l10n').t('Oops something went wrong. Please try again'));
```

Comment:
How about showing the exact error message specially when the user is the admin ?